src/panels/lovelace/hui-root.ts

```diff
@@ -405,6 +411,11 @@ class HUIRoot extends LitElement {
     ];
   }
 
+  protected firstUpdated(changedProperties: PropertyValues) {
+    super.updated(changedProperties);
+    this._conversation = isComponentLoaded(this.hass, "conversation");
```

Comment:
This is not correct. We should check it on every render. If a user installs Almond, we want this icon to show up. You could memoize it though on `hass.config.components`.

Reply:
Yeah of course 🤦‍♂

Reply:
We might want to create a new `isComponentLoaded` that takes hass.config.components and so is better to memoize.

Reply:
Do we need to optimise it for states and shopping list or is checking it on every change of hass ok?

Reply:
I just care about Lovelace right now.